src/components/Checkbox/index.js

```diff
@@ -0,0 +1,78 @@
+import { Component } from 'pet-dex-utilities';
+import './index.scss';
+
+const events = ['change', 'text:change', 'disable', 'value:change'];
+
+const html = `
+    <label class="checkbox-container">
+        <input data-select="checkbox" type="checkbox" value="">
+        <span data-select="checkbox-text" class="checkmark"></span>
+    </label>
+`;
+
+export default function Checkbox({ isChecked = false, name = '', text = '', isDisabled = false, value = '' } = {}) {
+  Component.call(this, { html, events });
+
+  this.setChecked(isChecked);
+  this.setText(text);
+  this.setName(name);
+  this.setDisabled(isDisabled);
+  this.setValue(value);
+
+  const $checkbox = this.selected.get('checkbox');
+
+  $checkbox.addEventListener('change', (e) => {
+    const isRealChecked = e.target.checked;
+    this.setChecked(isRealChecked);
```

Comment:
Po coloca isso numa linha só 
```suggestion
    this.setChecked(e.target.checked);
```